doc/ref_cert/lfn/chapters/chapter01.md

```diff
@@ -26,8 +46,28 @@ This document covers the realisation aspects of Verification and Certification o
 - Gap analysis to identify where the Gaps are in the industry (tooling, test cases, process, etc).
 - Identify development efforts needed to address any gaps identified.
 
+**Not in Scope**
+- Functional testing/validation of the VNF is not in scope.
```

Comment:
Should this be "Functional testing / validation of the application provided by the VNF is outside the scope of this work (i.e. testing of security capabilities provided by a VNF firewall would be outside of the scope).